explore_warming.ipynb

```diff
@@ -290,12 +288,56 @@
     "    ax.scatter(power_plants.lon, power_plants.lat, transform=ccrs.PlateCarree(),s=0.2,c='k')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "93761342-6d30-46a8-ad7d-f70c6fdeb679",
+   "metadata": {},
+   "source": [
+    "## Step 3: Export\n",
+    "\n",
+    "Use the below code to export a dataset as a NetCDF, GeoTIFF, or CSV file. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b428626f-e9a2-4f4f-aa51-7dc924be2fad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.export_as()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be14a5bf-b766-4a43-9ed5-16c0737c980c",
+   "metadata": {},
+   "source": [
+    "Provide the name of the dataset in the environment to export as well as a character string containing the file name in quotations. \n",
+    "\n",
+    "If the dataset contains multiple variables, provide an argument specifying which variable to export (e.g. variable=”T2”). \n",
+    "\n",
+    "If you would like to save data as a GeoTIFF or CSV file and the dataset contains scenarios or simulations, additionally provide arguments specifying the scenario (scenario=”historical”) and the simulation (simulation=”cesm2”)."
+   ]
+  },
```

Comment:
Similarly, I'm not sure if you can do this anymore. I would recommend consulting Beth.

Reply:
This will need to be modified in getting_started.ipynb as well.